packages/template-ui/vue.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   publicPath: './',
   transpileDependencies: ['vue-clamp', 'resize-detector'],
+  css: {
+    extract: false,
```

Comment:
This is unfortunate, as removing the standalone CSS file makes the JS huge. But on the other hand is great that you found the source of the error.
[This issue](https://github.com/vuejs/vue-cli/issues/4646#issuecomment-538605177) marks mini-css-extract-plugin "^0.6.0" as rotten. We have 2 versions, 0.4.5 and 0.9.0. The older because of kolibri-tools:

```
$ yarn why mini-css-extract-plugin
yarn why v1.22.10
[1/4] Why do we have the module "mini-css-extract-plugin"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "mini-css-extract-plugin@0.4.5"
info Reasons this module exists
   - "_project_#discovery-page#kolibri-tools" depends on it
   - Hoisted from "_project_#discovery-page#kolibri-tools#mini-css-extract-plugin"
info Disk size without dependencies: "68KB"
info Disk size with unique dependencies: "396KB"
info Disk size with transitive dependencies: "3.63MB"
info Number of shared dependencies: 12
=> Found "@vue/cli-service#mini-css-extract-plugin@0.9.0"
info This module exists because "_project_#pbskids-ui#@vue#cli-service" depends on it.
info Disk size without dependencies: "100KB"
info Disk size with unique dependencies: "468KB"
info Disk size with transitive dependencies: "3.8MB"
info Number of shared dependencies: 19
```

We should at least add a comment here explaining that this is because of IE11.

Reply:
Good idea, with the comment we can remove this once we can get rid of IE. I don't know exactly why this is failing, because the rules are in the css file, but who knows.